scripts/npm/prepublish/lib/build.js

```diff
@@ -1,11 +1,16 @@
-const exec = require( 'child-process-promise' ).exec;
+const proc = require( 'child-process-promise' );
```

Comment:
Not so into the name `proc` for this. Process is a very generic name, `childProcess` would be better.

Reply:
Good point. I still need to pair with @contolini and see if we really even need to require the entire package or if we can get the tests working with just `exec`

Reply:
Yeah I see this is used a bunch, so no worries if it's too much of a hassle for now.

Reply:
Fixed